src/lightning/fabric/fabric.py

```diff
@@ -593,20 +593,20 @@ def sharded_model(self) -> Generator:
             yield
 
     @contextmanager
-    def init(self) -> Generator:
-        """Instantiate under this context manager to apply improvements based on your configuration.
+    def init_tensor(self) -> Generator:
```

Comment:
better not change lightning existing APIs.

Reply:
yes since these are upstream changes , may be we can keep them off for now .